library/std/src/f128.rs

```diff
@@ -489,6 +491,13 @@ impl f128 {
     /// assert!(abs_difference <= f128::EPSILON);
     /// # }
     /// ```
+    ///
+    /// Non-positive values:
+    /// ```
+    /// #![feature(f128)]
+    /// assert_eq!(0_f128.ln(), f128::NEG_INFINITY);
+    /// assert!((-42_f128).ln().is_nan());
+    /// ```
```

Comment:
Since `f128` math is not available on all platforms yet, these need to be wrapped in a config like the other examples:

```rust
    /// #![feature(f128)]
    /// # #[cfg(reliable_f128_math)] {
    ///
    /// // example here
    ///
    /// # }
```